main.js

```diff
@@ -0,0 +1,354 @@
+
+  // Checks if it is human vs. computer or player vs. player
+  else if (status.textContent === `Player: ${playerOneName}'s turn!` || status.textContent === "Human's turn!") {
+    // On click place an X
+    event.target.textContent = 'X';
+
+    // Change to player two if two player game is clicked
+    if (playerTwoName !== '') {
+      status.textContent = `Player: ${playerTwoName}'s turn!`;
+    }
+    // Change to computer's turn if one player game is clicked
+    else {
+      status.textContent = "Computer's turn!";
+    }
+
+    //Checks if after click, player one wins on two player
+    winCheck();
+    if (winCheck() === true) {
+      if (status.textContent !== "Human's turn!") {
+        status.textContent = `Player: ${playerOneName} has won!`;
+      }
+      // Checks if after click, human wins
+      else {
+        status.textContent = 'The human wins!';
+      }
+    }
+  }
+
+  //If Player Two's turn ---------------------------------------------------------------------------------------------------------------------------------
+  else if (status.textContent === `Player: ${playerTwoName}'s turn!`) {
+    // On click place an O
+    event.target.textContent = 'O';
+
+    // Change to Player One
+    status.textContent = `Player: ${playerOneName}'s turn!`;
+
+    //Checks if after click, player two wins
+    winCheck();
+    if (winCheck() === true) {
+      status.textContent = `Player: ${playerTwoName} has won!`;
+    }
+  }
+
+  //If Computer's turn -----------------------------------------------------------------------------------------------------------------------------------
+  else if (status.textContent === "Computer's turn!") {
+    computerAi();
+    winCheck();
+    if (winCheck() === true) {
+      status.textContent = 'The Computer Wins!';
+    }
+  }
+}
+
+//Checks to see if there is a winner ---------------------------------------------------------------------------------------------------------------------
+function winCheck() {
+  //for every index in every element in the array created by winCombos keys
+  for (let combo of Object.values(winCombos)) {
+    //Does not set off alert when all cells are blank
+    if (combo[0].textContent !== '') {
+      //If they are all equal the same thing
+      if (combo[0].textContent === combo[1].textContent && combo[1].textContent === combo[2].textContent) {
+        // Strike a line through the characters
+        combo[0].style.setProperty('color', 'gold');
+        combo[1].style.setProperty('color', 'gold');
+        combo[2].style.setProperty('color', 'gold');
+        clearInterval(timeInterval);
+        return true;
+      }
+    }
+  }
+}
+
+// Game Timer --------------------------------------------------------------------------------------------------------------------------------------------
+function gameTimer() {
```

Comment:
This is a good function, and a fine way to format your timer display, but you could make it a bit more concise by using math instead of logic (or maybe a little of both). the `%` operator might be very useful here.